spec/features/staff/users_can_view_an_activity_as_xml_spec.rb

```diff
@@ -133,14 +133,14 @@
           create(:project_activity,
             organisation: organisation,
             delivery_partner_identifier: "IND-ENT-IFIER",
-            policy_marker_gender: "not_targeted",
-            policy_marker_biodiversity: "significant_objective",
-            policy_marker_disability: "principal_objective",
-            policy_marker_desertification: "principal_objective_and_in_support",
-            policy_marker_nutrition: "not_assessed",
-            policy_marker_climate_change_adaptation: "not_assessed",
-            policy_marker_climate_change_mitigation: "not_assessed",
-            policy_marker_disaster_risk_reduction: "not_assessed")
+            policy_marker_gender: 0,
```

Comment:
It would likewise be useful to use meaningful names rather than magic numbers in these definitions and in tests.